weblate/trans/tests/test_views.py

```diff
@@ -399,6 +400,42 @@ def test_home_without_whiteboard(self):
         response = self.client.get(reverse('home'))
         self.assertNotContains(response, 'whiteboard')
 
+    def test_component_list(self):
+        clist = ComponentList(name="TestCL", slug="testcl")
+        clist.save()
+
+        response = self.client.get(reverse('home'))
+        self.assertContains(response, 'TestCL')
+        self.assertEquals(len(response.context['componentlists']), 1)
```

Comment:
Please use assertEqual, the one with s is deprecated for quite some time (and in Python 3 it can cause noisy warnings).